pkg/verify/signed_entity.go

```diff
@@ -221,6 +223,26 @@ func NewVerificationResult() *VerificationResult {
 	}
 }
 
+// MarshalJSON deals with protojson needed for the Statement.
+// Can be removed when https://github.com/in-toto/attestation/pull/403 is merged.
+// TODO: Should we also add UnmarshalJSON?
```

Comment:
If someone were to try to parse the json ouput using Go,

- would they use this library's types to do so?
- would the unmarshal step work with the output sans errors?

if it would error out otherwise, seems wise to provide a working output/input loop by default (i.e. yes to adding an UnmarshalJSON)?

Reply:
I'm just being lazy. It's easy enough to add, so I'll do it.